tests/detection.cc

```diff
@@ -0,0 +1,62 @@
+/*
+ *    Copyright 2022 Cryspen Sarl
+ *
+ *    Licensed under the Apache License, Version 2.0 or MIT.
+ *    - http://www.apache.org/licenses/LICENSE-2.0
+ *    - http://opensource.org/licenses/MIT
+ */
+
+#include <fstream>
+#include <gtest/gtest.h>
+
+#include "EverCrypt_AutoConfig2.h"
+#include "hacl-cpu-features.h"
+#include "util.h"
+
+using namespace std;
+
+class DetectionSuite : public ::testing::Test
+{};
+
+TEST(DetectionSuite, PrintFeatures)
+{
+  // HACL
+  {
+    cout << "# HACL" << endl;
+    hacl_init_cpu_features();
+
+#ifdef HACL_CAN_COMPILE_VEC128
+    cout << "HACL_CAN_COMPILE_VEC128: " << 1 << endl;
+    cout << "hacl_vec128_support(): " << hacl_vec128_support() << endl;
+#else
+    cout << "HACL_CAN_COMPILE_VEC128: " << 0 << endl;
+#endif
+
+#ifdef HACL_CAN_COMPILE_VEC256
+    cout << "HACL_CAN_COMPILE_VEC256: " << 1 << endl;
+    cout << "hacl_vec256_support(): " << hacl_vec128_support() << endl;
```

Comment:
Fixme

Reply:
Fixed in [ab8d013](https://github.com/cryspen/hacl-packages/pull/150/commits/ab8d01377e9d86f3f9fdb502493cb9b0f3c6d6df).